src/ui/public/vislib/lib/handler.js

```diff
@@ -62,7 +62,7 @@ export function VisHandlerProvider(Private) {
 
       this.renderArray = this.renderArray
         .concat(this.valueAxes)
-        .concat(this.categoryAxes);
+        .concat(this.categoryAxes.slice().reverse());
```

Comment:
Do you think it'd be overkill to add a comment in here, maybe with a link to the issue?  If I was reading this as a developer I'd have no idea why the axes were reversed.